src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -167,14 +167,15 @@ private void initUI() {
 
     private ScrollPane createAllMilestonesScrollPane() {
         VBox milestonesBox = new VBox();
-        milestonesBox.setStyle("-fx-border-radius: 3;-fx-background-color: white;-fx-border-color: black;");
+        milestonesBox.setPrefWidth(398);
+        milestonesBox.setStyle("-fx-background-color: white;");
         originalMilestones.forEach(milestone ->
                 milestonesBox.getChildren().add(
                 setMouseClickForNode(milestone.getDetailedMilestoneNode(), milestone.getTitle()))
         );
 
         ScrollPane milestonesScrollPane = new ScrollPane();
-        milestonesScrollPane.setMaxHeight(163);
+        milestonesScrollPane.setMaxHeight(160);
```

Comment:
Actually, could this value be the same as the `Search results`'s i.e. set to `height * no. of milestones`?